src/App.tsx

```diff
@@ -1,7 +1,18 @@
-import './App.css'
+import './App.css';
```

Comment:
We have prettierrc.js file in which we have rules for prettier formatter. 
Install this formatter and when you save files it should auto format your code.
<img width="1437" alt="Screenshot 2024-02-26 at 14 12 28" src="https://github.com/SierraCSWebDev/WebsiteProject/assets/93590594/9d3e82bf-8cce-447d-b959-6d7383081a66">

If prettier is not formatting anything you need to go to your settings and check format on save, or check the prettier settings.
<img width="1208" alt="Screenshot 2024-02-26 at 14 17 11" src="https://github.com/SierraCSWebDev/WebsiteProject/assets/93590594/24c1ce65-2bd6-4d82-a2b6-a7efbea56a82">